### DISCUSSION TITLE
Chromebook and DDEV-Local Blog article update request

### DISCUSSION THREAD

**Discussion by ops-andy** [ddev/ddev#2740] [General]
@rfay per your request, migrating this to here from Gitter.
 I got a new Chromebook specifically so I could run Docker and do more actual work on my Chromebook (my old Chromebook was never going to get the Crostini support).
I read your article here: https://www.ddev.com/ddev-local/ddev-docker-chromebook/ which was great to get started.
A few things I'd add:
* I suggest adding a sudo password sudo passwd [username] . I had issues installing Linuxbrew because by default there is no password set, but it kept asking for a sudo password for some reason. Once I re-installing using a sudo password, I had no issues. This could be a just me type issue.
* you mentioned using the LiveHosts chrome ext. This worked ok. But even better was just installing Firefox in the Linux container. By doing so, Firefox runs in the container and thus doesn't need any redirects for the domains. Just install via flatpack (https://support.mozilla.org/en-US/kb/run-firefox-chromeos). The biggest reason to do this is that I can now access sites via https in Firefox.
Other than I got Docker and DDEV running without issue. Thanks for the guide.

Chromebook: Acer Spin 713
CPU: Intel i5 10210U
Ram: 8GB
HD: 128GB SSD

**Comment by rfay** [comment]
Thanks @ops-andy - it's been more than a year since I had my hands on a Chromebook. 

* At what step should the `sudo passwd $USER` be done?
* It makes sense to install firefox (and probably PHPStorm too right) in the container and just not deal with the whole https problem.

**Reply by ops-andy** [reply]
- I would add the sudo password right after you first open the Linux terminal and connect to the container. So that would be #2 in your guide.
- Yes. I'm not sure what you lose by installing apps in the container yet. I think you lose stylus support, and maybe some of the system notifications, etc that you would normally have. Not 100% sure yet. However, for web dev apps I'm always going to be working in the container so I don't think I have a need for them to be outside the container.

**Comment by rfay** [comment]
I added your suggestions to https://www.ddev.com/ddev-local/ddev-docker-chromebook/ - would appreciate your review. Thanks @ops-andy !

**Reply by ops-andy** [reply]
looks good!
